sieve.js

```diff
@@ -366,35 +369,34 @@
             var type = null;
             var params = null;
 
-            switch (element.Type)
-            {
-                case "Reject":
+            switch (element.Type) {
+                case 'Reject':
                     throw { name: 'UnsupportedRepresentation', message: 'Unsupported filter representation: Reject' };
 
-                case "Redirect":
+                case 'Redirect':
                     throw { name: 'UnsupportedRepresentation', message: 'Unsupported filter representation: Redirect' };
 
-                case "Keep":
+                case 'Keep':
                     break;
 
-                case "Discard":
-                    actions.FileInto.push("trash");
+                case 'Discard':
+                    actions.FileInto.push('trash');
                     break;
 
-                case "FileInto":
+                case 'FileInto':
                     var name = element.Name;
                     actions.FileInto.push(name);
                     break;
 
-                case "AddFlag":
-                    type = "mark";
+                case 'AddFlag':
+                    type = 'mark';
 
-                    var read = (element.Flags.indexOf("\\Seen") >= 0);
-                    var starred = (element.Flags.indexOf("\\Flagged") >= 0);
+                    var read = (element.Flags.indexOf('\\Seen') >= 0);
+                    var starred = (element.Flags.indexOf('\\Flagged') >= 0);
 
                     actions.Mark = {
-                        "Read": read,
-                        "Starred": starred
+                        Read: read,
+                        Starred: starred
                     };
                     break;
 
```

Comment:
We should add a `case 'Vacation':` here to reverse the parsing from the `Tree` to the simple format.

Reply:
Done.